From 5cac8da636e24a9394d3e2fe250afa19e4c898ee Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Sun, 10 Sep 2017 20:22:03 -0700
Subject: [PATCH 4/4] lineage: Enable wifi wakeup config option

Enables the setting to have wifi turn on automatically
when you're near high quality saved networks.

Change-Id: Id61c3f5c655322e54c015f55b6f389b835c27fe5
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index 08e1c8d..a3797b0 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -85,4 +85,10 @@
    <!-- Set icon mask to circle -->
    <string name="config_icon_mask" translatable="false">"M50 0A50 50,0,1,1,50 100A50 50,0,1,1,50 0"</string>
 
+    <!-- Controls the WiFi wakeup feature.
+          0 = Not available.
+          1 = Available.
+    -->
+    <integer translatable="false" name="config_wifi_wakeup_available">1</integer>
+
 </resources>
-- 
2.7.4

